tests/common.sh

```diff
@@ -95,7 +95,7 @@ get_receipt() {
   set +e
 
   while true; do
-    output=$(../bin/aergocli receipt get --port $query_port $txhash 2>&1 > receipt.json)
+    output=$(../bin/aergocli receipt get $txhash 2>&1 > receipt.json)
```

Comment:
Does this guarantee connection to the same node? I suspect that some real change is missing in this PR.

Reply:
by removing this, it will connect to the default port
this is what happens on other parts of the integration test
so all queries will all be made to the same port, and thus to the same node (because each node uses a different port)